notification/docs/DevelopmentGuide.md

```diff
@@ -1,45 +1,27 @@
+ - **Node.js** version 12.
+ - **Npm** version 6.
+ 
+You can install all dependencies using `npm` with following command:
 
-## Requirements
-Node.js version 12 LTS or higher is supported.
+```
+npm install
+```
 
-## Run Module
-Execute `npm сi` to download all dependencies.
+## Development
+While developing project you can use some predefined commands for running tests, running linter or generating coverage. 
+ 
+- Execute `npm run test` to run all tests and print the code coverage report.
+- Execute `npm run unit` to run [Unit tests](../test/unit)
+- Execute `npm run integration` to run [Integration tests](../test/integration) 
+- Execute `npm run lint` to show lint errors in the code.
 
-In order to run the notification module, you have to provide following environmental variables:
-
-Name | Content | Required | Default value
------------- | ------------- | ------------- | -------------
-CTP_PROJECT_KEY | commercetools project key (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
-CTP_CLIENT_ID | commercetools client ID with `manage_types` and `manage_payments` scopes (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
-CTP_CLIENT_SECRET | commercetools client secret (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
-CTP_HOST | commercetools HTTP API is hosted at that URL| NO | https://api.europe-west1.gcp.commercetools.com
-CTP_AUTH_URL | commercetools’ OAuth 2.0 service is hosted at that URL | NO | https://auth.europe-west1.gcp.commercetools.com
-ADYEN_ENABLE_HMAC_SIGNATURE | Verify the integrity of notifications using [Adyen HMAC signatures](https://docs.adyen.com/development-resources/webhooks/verify-hmac-signatures) | NO | `true`
-ADYEN_SECRET_HMAC_KEY | The generated secret HMAC key that is linked to a Adyen **Standard Notification** endpoint | NO | 
-LOG_LEVEL | bunyan log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`)| NO | `info`
-PORT | port on which the application will run | NO | 443
-KEEP_ALIVE_TIMEOUT | milliseconds to keep a socket alive after the last response ([Node.js docs](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_server_keepalivetimeout)) | NO | Node.js default
-ENSURE_RESOURCES | Set to `false` to disable the creation of resources in commercetools (e.g. custom types) | NO | `true`
-
-Don't forget to provide all required environment variables
-After setting all variables, execute command `npm run start` to run the module.
-
-## Run Tests
-
-- Execute `npm run unit-test` to run [Unit tests](../test/unit)
-- Execute `npm run integration-test` to run [Integration tests](../test/integration) (all required environment variables must be set)
-
-Execute `npm run test` to run all tests
-
-In the end, a code coverage report will be printed.
+> You need to set [environment variables](./DeploymentGuide.md#environment-variables) to be able to run integration tests.
```

Comment:
Does it work with any empty CT project or you need a special / preconfigured one?

Reply:
Yes, tests will bootstrap the required resources. 

But I have checked again it' like we did not adopt the IT of notification with the new payment types, I will try to update those too 🤦 https://github.com/commercetools/commercetools-adyen-integration/blob/355-notification-docs/notification/test/resources/payment-custom-type.json

Reply:
The issue is created, I will handle this as a separate PR: https://github.com/commercetools/commercetools-adyen-integration/issues/365